index.md

```diff
@@ -82,11 +82,10 @@ differently in Sopel. The vast majority of modules should work without problems,
 though.</dd>
 
 <dt>Can I use Sopel with Docker?</dt>
-<dd>Probably, but there's not much reason to. We only provide support for one
-version of Sopel at a time, and it already supports running multiple instances
-side-by-side. If you really need to containerize dependencies, virtualenv is
-probably a lot easier—but there is <a href="https://github.com/sopel-irc/docker-sopel">an
-Officially Unofficial™ Docker container</a> available.</dd>
+<dd>Yes, there is <a href="https://github.com/sopel-irc/docker-sopel">an
```

Comment:
> dd>Yes, there

Dagummit, I had that comma there and switched it just seconds before committing :sweat_smile:. I like the changes, though. :+1:

Reply:
lmao, I'm so bleary-eyed I thought the `.` was just an artifact. 😂

Glad you like my rewrite. I'll merge this when I have time to immediately rebase #26 onto it. Not that that would take long, but I'm already supposed to have gone to bed half an hour ago! 😅